assignments/callbacks.js

```diff
@@ -19,30 +19,85 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     console.log(first)
   });
 
-*/
+
 
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+return cb(arr.length);
+
 }
+getLength(items, function(length) {
+console.log(length)
+});
+
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
-}
 
+  return cb(items.length - 1);
```

Comment:
you want to be passing in the last item of the array, but here you are passing in a value which will be the array's length minus one, and you'll be using the `arr` variable since that is how it's being referred to in the function's parameters. remember to use bracket notation to access this like `arr[arr.length-1]`